system/systemtest.mk

```diff
@@ -50,6 +50,6 @@ define SYSTEMTEST_CMD_TEMPLATE
 perl $(SYSTEMTEST_RESROOT)$(D)stf$(D)stf.core$(D)scripts$(D)stf.pl \
 	-test-root=$(Q)$(SYSTEMTEST_RESROOT)$(D)stf;$(SYSTEMTEST_RESROOT)$(D)openjdk-systemtest$(OPENJ9_PRAM)$(Q) \
 	-systemtest-prereqs=$(Q)$(SYSTEMTEST_RESROOT)$(D)systemtest_prereqs$(Q) \
-	-java-args=$(Q)$(JVM_OPTIONS)$(Q) \
+	-java-args=$(SQ)$(JVM_OPTIONS)$(SQ) \
```

Comment:
I assume we can enforce JVM_OPTIONS to use single quotes?

Reply:
@llxia do you think this will work for the special jit options?

Reply:
I tested some JIT options and it worked, but it is hard to say that it will work for all of them. There are have some complex JIT options. Also, `JVM_OPTIONS ` is used without quotes in most cases. For example,
https://github.com/AdoptOpenJDK/openjdk-tests/blob/c6f016dbe374fa2f669b4cda77bfffcaa2c1702a/functional/MBCS_Tests/datetime/playlist.xml#L21

The system test is a special case, where we need to pass `JVM_OPTIONS` into STF. My personal preference is not to have quotes in `JVM_OPTIONS` and only add quotes in this special case.

Reply:
Sure, I will keep this change to add single quotes in the system mk file only.